src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -504,6 +504,34 @@ public class EnumeratorCancellationAttribute : Attribute
 }
 ";
 
+        protected const string NativeIntegerAttributeDefinition =
+@"using System.Collections.Generic;
+namespace System.Runtime.CompilerServices
+{
+    [System.AttributeUsage(
+        AttributeTargets.Class |
+        AttributeTargets.Event |
+        AttributeTargets.Field |
+        AttributeTargets.GenericParameter |
+        AttributeTargets.Parameter |
+        AttributeTargets.Property |
+        AttributeTargets.ReturnValue,
+        AllowMultiple = false,
+        Inherited = false)]
+    public sealed class NativeIntegerAttribute : Attribute
+    {
+        public NativeIntegerAttribute()
+        {
+            TransformFlags = new[] { true };
+        }
+        public NativeIntegerAttribute(bool[] flags)
+        {
+            TransformFlags = flags;
+        }
+        public readonly IList<bool> TransformFlags;
```

Comment:
>IList<bool> [](start = 24, length = 11)

Just curious why we are using IList. Are we using an existing type as a template?

Reply:
Yes, this matches DynamicAttribute.

---
In reply to: [372671234](https://github.com/dotnet/roslyn/pull/41160#discussion_r372671234) [](ancestors = 372671234)